lib/shopify_api/resources/vault_session.rb

```diff
@@ -0,0 +1,28 @@
+module ShopifyAPI
+  module VaultSessionFormat
```

Comment:
That doesn't really address why you need the change. Why is that endpoint behaving differently then other places?